app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -84,3 +99,7 @@ data class SelfUserData(
     val avatarAsset: UserAvatarAsset? = null,
     val status: UserAvailabilityStatus = UserAvailabilityStatus.NONE
 )
+
+sealed class HomeSnackBarState(private val randomEventIdentifier: UUID) {
```

Comment:
I don't know what this `randomEventIdentifier` is for, do we need it here?

Also, small suggestion - maybe it would be better to add `None` type instead of operating on nullable type 🤔

Reply:
I also suspect that might be a leftover/unused code? So it could be removed as we are not really using it at all.

And I also agree that having a `None` status instead of null would read slightly better.

Reply:
According to other place that we have `randomIdentifier` `We are adding a [randomEventIdentifier] as [UUID], so the msg can be discarded every time after being generated.`

Reply:
I will add None type

Reply:
Actually I just realised that if we add `None` type, then we will not be able to filter those null states on the `HomeScreen.handleSnackBarMessage()`, so leaving them as null for now until we unify all these snackbar messages across screens.

Reply:
I think we can simply replace `?.` with `when` and show snackbar when `HomeSnackbarState !is HomeSnackbarState.None` instead of showing snackbar when `HomeSnackbarState != null`